src/main/java/uk/gov/justice/digital/delius/config/FeatureSwitches.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.delius.config;
+
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "features")
+@Data
+public class FeatureSwitches {
```

Comment:
TODO: I would like to refactor the switches properties to get rid of the nasty nesting. That will be a future PR